webapp/store/logic.py

```diff
@@ -360,11 +361,12 @@ def get_video_embed_code(url):
 
 
 def filter_screenshots(media):
+    banner_regex = r"/banner(\-icon)?(_.*)\.(png|jpg)"
     return [
         m["url"]
         for m in media
-        if m["type"] == "screenshot" and "banner" not in m["url"]
-    ]
+        if m["type"] == "screenshot" and not re.search(banner_regex, m["url"])
+    ][:5]
```

Comment:
so here you get only the first 5 elements of the array?

Reply:
:+1: